### PR TITLE
Webhooks 1.1 release notes and update

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) creates release notes and upgrade information for the 1.1.0 release of Webhooks.

## Affected pages

- ...

## Links to Magento Open Source code

- ...

### CODE REVIEW

**Review by jhadobe — APPROVED** [review]

File: src/pages/webhooks/release-notes.md [hunk 1]
```diff
@@ -9,6 +9,28 @@ keywords:
...
+
+### Enhancements
+
+* You can now define [conditional webhooks](conditional-webhooks.md) that run only when configured conditions are met. <!--- CEXT-2541 -->
+
+* Prior to this release, the only supported webhook response values were `success` and `exception`. This release adds support for `add`, `remove`, and `replace` [responses](responses.md#responses). <!--- CEXT-2405, 2413, 2468 -->
+
+* You can now [configure specific fields within an array](hooks.md#configure-hooks) to transmit in a webhook. <!--- CEXT-2521 -->
+
+* Added the `webhooks_response` cache type. <!--- CEXT-2403 -->
```
Comment (oshmyheliuk) [inline src/pages/webhooks/release-notes.md:26]:
```
Added a possibility to cache webhook responses and specify cache duration. Added a new cache type to store webhook responses.
```

File: src/pages/webhooks/release-notes.md [hunk 2]
```diff
@@ -9,6 +9,28 @@ keywords:
...
+
+### Release date
+
+November 30, 2023
+
+### Enhancements
+
+* You can now define [conditional webhooks](conditional-webhooks.md) that run only when configured conditions are met. <!--- CEXT-2541 -->
+
+* Prior to this release, the only supported webhook response values were `success` and `exception`. This release adds support for `add`, `remove`, and `replace` [responses](responses.md#responses). <!--- CEXT-2405, 2413, 2468 -->
```
Comment (oshmyheliuk) [inline src/pages/webhooks/release-notes.md:22]: I would rephrase it to something like:
```
Added support for data-modifying webhooks by introducing new response operations: `add`, `remove`, and `replace`
```

File: src/pages/webhooks/release-notes.md [hunk 3]
```diff
@@ -9,6 +9,28 @@ keywords:
...
+
+* You can now [configure specific fields within an array](hooks.md#configure-hooks) to transmit in a webhook. <!--- CEXT-2521 -->
```
Comment (oshmyheliuk) [inline src/pages/webhooks/release-notes.md:24]: I would skip this one from the release notes as it was an improvement for the new `add` operation

**Review by oshmyheliuk — APPROVED** [review]
